client/components/filters/advanced/index.js

```diff
@@ -100,7 +100,7 @@ class AdvancedFilters extends Component {
 			newFilter.rule = filterConfig.rules[ 0 ].value;
 		}
 		if ( filterConfig.input && filterConfig.input.options ) {
-			newFilter.value = filterConfig.input.options[ 0 ].value;
+			newFilter.value = filterConfig.input.defaultOption || filterConfig.input.options[ 0 ].value;
```

Comment:
Consider future proofing this by logging a warning if defaultOption is not in options.